parse.go

```diff
@@ -66,6 +74,13 @@ func Parse(in io.Reader, bytesToRead int64, frameChan chan *frame.Frame, opts ..
 	for !p.reader.rawReader.IsLimitExhausted() {
 		_, err := p.Next()
 		if err != nil {
+			if err.Error() == "EOF" {
```

Comment:
```suggestion
			if errors.Is(err, io.EOF) {
```

I think you should be able to do the above.